public/app/features/logs/components/panel/InfiniteScroll.tsx

```diff
@@ -132,24 +136,40 @@ export const InfiniteScroll = ({
     ({ index, style }: ListChildComponentProps) => {
       if (!logs[index] && infiniteLoaderState !== 'idle') {
         return (
-          <LogLineMessage style={style} onClick={infiniteLoaderState === 'pre-scroll' ? onLoadMore : undefined}>
+          <LogLineMessage
+            style={style}
+            styles={styles}
+            onClick={infiniteLoaderState === 'pre-scroll' ? onLoadMore : undefined}
+          >
             {getMessageFromInfiniteLoaderState(infiniteLoaderState, sortOrder)}
           </LogLineMessage>
         );
       }
       return (
         <LogLine
+          displayedFields={displayedFields}
           index={index}
           log={logs[index]}
           showTime={showTime}
           style={style}
+          styles={styles}
```

Comment:
Nit: Looks like these styles aren't dependent on the state of the parent, maybe get rid of this prop and define the styles directly in the log line?

Reply:
The idea is to have this called once and then be reused for every log line. Even if useStyles2 is memoized, is just one less function call to render a log line.